webapp/src/Service/CheckConfigService.php

```diff
@@ -479,9 +479,15 @@ public function checkContestsValidate(): ConfigCheckItem
 
         $desc = '';
         foreach ($contesterrors as $cid => $errors) {
-            $desc .= "Contest: c$cid: " .
-                /* @phpstan-ignore-next-line */
-                    (count($errors) == 0 ? 'no errors' : (string)$errors) ."\n" .$cperrors[$cid];
+            $desc .= "Contest: c$cid: ";
+            if (count($errors) == 0) {
+                $desc .= "no errors\n";
+            } else {
+                $desc .= "errors:\n";
```

Comment:
```suggestion
                $desc .= "errors:\n" . ($cperrors[$cid] ? $cperrors[$cid] . '\n' : '');
```

Did not check if it works, but as you mentioned last night the $cperrors are missing.

Reply:
Also added it to the if statement above and there is no newline necessary as it adds it already